arm-sql/2015-05-01-preview/examples/DatabaseAdvisorGet.json

```diff
@@ -0,0 +1,25 @@
+{
+    "parameters": {
+        "subscriptionId": "741fd0f5-9cb8-442c-91c3-3ef4ca231c2a",
```

Comment:
These examples will show up on https://docs.microsoft.com , I suggest replacing real subscription id's with fake ones like in other examples